app/src/org/commcare/utils/FileExtensionNotFoundException.java

```diff
@@ -0,0 +1,10 @@
+package org.commcare.utils;
+
+/**
+ * @author $|-|!˅@M
+ */
+public class FileExtensionNotFoundException extends RuntimeException {
```

Comment:
Since we are catching this in every case, I think we should make this a typed IOException to catch it in common IO exception handling blocks.